community/settings.py

```diff
@@ -36,6 +36,7 @@
     'gci',
     'gsoc',
     'data',
+    'org',
```

Comment:
As I already explained at https://github.com/coala/community/issues/102#issuecomment-373075029 , 'org' != git.  It also doesnt equal IGitt.

For more about 'org', see new issue https://github.com/coala/community/issues/113